service/src/services/BaseService.ts

```diff
@@ -63,6 +63,21 @@ async function uploadResourcesFromBundle(entries: DetailedEntry[]) {
  */
 async function insertBundleResources(entry: DetailedEntry) {
   if (entry.resource?.resourceType === 'Library' || entry.resource?.resourceType === 'Measure') {
+    if (entry.resource.status != 'active') {
```

Comment:
Do we want this for any Measure or Library we insert in the database? Not just the ones from a bundle?

Reply:
I think that Hoss confirmed yes on that at dev hour.

Reply:
I added code to coerce to active for a regular create/update (non-bundle). Tried a couple of different approaches to get a similar OperationOutcome, but because we're not creating responses in the bundle it's a bit trickier, and my approaches messed up other things that were a part of the base functionality. I have those attempts locally stashed, so happy to discuss further, or just let it be with the logged warning rather than the operation outcome.